homework/shelkovich_ilya/product/src/test/java/com/tutrit/java/quickstart/service/ProductServiceTest.java

```diff
@@ -0,0 +1,30 @@
+package com.tutrit.java.quickstart.service;
+
+import com.tutrit.java.quickstart.bean.Product;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+public class ProductServiceTest {
+
+    Product product;
+    ProductService productService;
+    double regularPrice;
+    double discount;
+
+    @Before
+    public void setUp() {
+        this.product = new Product("water");
+        this.productService = new ProductService();
+        this.regularPrice = 10;
+        this.discount = 0.4;
```

Comment:
This could be moved directly to test. 
No need to instantiate it for every test.

Reply:
improved it